src/things_mcp/preflight.py

```
@@ -0,0 +1,63 @@
+    except ImportError as e:
+        errors.append(f"things.py not installed: {e}")
+
+    # 3. MCP SDK
+    try:
+        import mcp  # noqa: F401
+    except ImportError as e:
+        errors.append(f"MCP SDK not installed: {e}")
+
+    # 4. Things app reachable (macOS only)
+    try:
+        result = subprocess.run(
```

Ruff (0.15.6)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)